core/src/test/java/com/xmartlabs/base/core/util/DummyObject.java

```diff
@@ -0,0 +1,15 @@
+package com.xmartlabs.base.core.util;
+
+import lombok.Builder;
+import lombok.Data;
+
+/**
+ * Created by mike on 11/04/2017.
+ */
+@Data
+@Builder
+public class DummyObject {
```

Comment:
If this is used only in one class maybe it can be present as a static inner class.